node/_core.ts

```diff
@@ -22,6 +22,9 @@ if (Deno?.core) {
     encode(chunk: string): Uint8Array {
       return new TextEncoder().encode(chunk);
     },
+    decode(chunk: Uint8Array): string {
+      return new TextDecoder().decode(chunk);
```

Comment:
I wonder how will the performance be impacted if the initialized text decoder will only be initialized once for the subsequent calls.

Reply:
This is a fallback implementation, but to answer your question: yes it will be slightly faster to inititalize once but only noticable for very small inputs in micro benchmarks.